.gitignore

```diff
@@ -1 +1,36 @@
-hosts.ini
+# ETC
+ansible.cfg
+hosts*
```

Comment:
I guess we can only keep these two, the rest should be in a global gitignore and are not specific to this project 👍